labs/CiscoContainerPlatform-101/1.md

```diff
@@ -97,6 +97,9 @@ Logging in presents you with the "Clusters" screen. Here we can see all the Kube
 
 ![](assets/images/ccp-clusters.jpeg)
 
+Note: CCP supports Single Master Node (v2) and Multi-master nodes (v3) deployment. Please choose v2 for the following steps.
```

Comment:
General note: all "Note:" lines should be written in the following format:
```
> *Note*: NOTE TEXT
```

Suggested rewrite for the first sentence: "CCP supports single-master node (v2) and multi-master nodes (v3) deployments."

Reply:
Updated 1.md